docs/usage/pod-security.md

```diff
@@ -13,3 +13,43 @@ in `spec.kubernetes.kubeAPIServer.admissionPlugins` field in the `Shoot` resourc
 Only if the `PodSecurityPolicy` admission plugin is disabled the cluster can be upgraded to `v1.25`.
 
 > :warning: You should disable the admission plugin and wait until Gardener finish at least one `Shoot` reconciliation before upgrading to `v1.25`. This is to make sure all the `PodSecurityPolicy` related resources deployed by Gardener are cleaned up.
+
+## Admission configuration for the `PodSecurity` admission plugin
+
+If you wish to add your custom configuration for the `PodSecurity` plugin, you can do so in the Shoot spec under `.spec.kubernetes.kubeAPIServer.admissionPlugins` by adding:
+```yaml
+admissionPlugins:
+- name: PodSecurity
+  config:
+    apiVersion: pod-security.admission.config.k8s.io/v1beta1
+    kind: PodSecurityConfiguration
+    # Defaults applied when a mode label is not set.
+    #
+    # Level label values must be one of:
+    # - "privileged" (default)
+    # - "baseline"
+    # - "restricted"
+    #
+    # Version label values must be one of:
+    # - "latest" (default) 
+    # - specific version like "v1.24"
+    defaults:
+      enforce: "privileged"
+      enforce-version: "latest"
+      audit: "privileged"
+      audit-version: "latest"
+      warn: "privileged"
+      warn-version: "latest"
+    exemptions:
+      # Array of authenticated usernames to exempt.
+      usernames: []
+      # Array of runtime class names to exempt.
+      runtimeClasses: []
+      # Array of namespaces to exempt.
+      namespaces: []
+```
+For proper functioning of Gardener, if not present, `kube-system` namespace will be added exempted to the exempted namespaces by Gardener.
+
+## `.spec.kubernetes.allowPrivilegedContainers` in the Shoot spec
+
+This field if set to `true` means that all authenticated users can use the "gardener.privileged" `PodSecurityPolicy`, allowing full unrestricted access to Pod features. But the `PodSecurityPolicy` admission plugin is removed in Kubernetes `v1.25` and `PodSecurity` has taken its place. Therefore, this field doesn't have any relevance in versions `>=1.25`. If you need to set a default pod admission level for your cluster, follow [Admission Configuration for the PodSecurity admission plugin](#admission-configuration-for-the-podsecurity-admission-plugin). 
```

Comment:
```suggestion
If this field is set to `true` then all authenticated users can use the "gardener.privileged" `PodSecurityPolicy`, allowing full unrestricted access to Pod features. However, the `PodSecurityPolicy` admission plugin is removed in Kubernetes `v1.25` and `PodSecurity` has is its successor. Therefore, this field doesn't have any relevance in versions `>= v1.25` anymore. If you need to set a default pod admission level for your cluster, follow [this documentation](#admission-configuration-for-the-podsecurity-admission-plugin). 
```

Reply:
Can you please rephrase, "However, the `PodSecurityPolicy` admission plugin is removed in Kubernetes `v1.25` and `PodSecurity` has is its successor" ? Did you mean, "has taken its place as its successor"?

Reply:
Sorry:

> However, the `PodSecurityPolicy` admission plugin is removed in Kubernetes `v1.25` and `PodSecurity` is its successor.

But your proposal is also fine for me.